### PR TITLE
Macpie/ch9751/after state channel outage miner sprays packets

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

**Comment by macpie** [comment]
Depends on https://github.com/helium/blockchain-core/pull/784

File: src/device/router_device_channels_worker.erl [hunk 1]
```diff
@@ -538,10 +541,8 @@ downlink_decode(Payload) ->
     Blockchain :: blockchain:blockchain()
 ) -> {ok, map()}.
 send_to_channel(CachedData0, Device, EventMgrRef, Blockchain) ->
-    FormatHotspot = fun(
-        #data_cache{pubkey_bin = PubKeyBin, packet = Packet, region = Region, time = Time}
-    ) ->
-        format_hotspot(Blockchain, PubKeyBin, Packet, Region, Time, <<"success">>)
+    FormatHotspot = fun(DataCache) ->
+        format_hotspot(DataCache, Blockchain)
```
Comment (michaeldjeffrey) [inline src/device/router_device_channels_worker.erl:545]: This worked out nicely 🎈

**Review by michaeldjeffrey — DISMISSED** [review]

File: src/device/router_device_channels_worker.erl (hunk 1 above)
Comment (macpie) [reply src/device/router_device_channels_worker.erl:545]: cleaner!

**Review by michaeldjeffrey — APPROVED** [review]
Version changes since last review. Still good

**Review by Vagabond — APPROVED** [review]

**Pull request merged**